Taken from: https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-libs/apr-util/files/apr-util-1.6.1-libtool.patch

https://bugs.gentoo.org/779487

From: orbea <orbea@riseup.net>
Date: Sat, 18 Jun 2022 20:20:09 -0700
Subject: [PATCH] configure: Add missing LT_INIT

This fixes the build with rlibtool which requires the generated libtool
to determine if building shared or static libraries.

--- a/configure.in
+++ b/configure.in
@@ -8,6 +8,8 @@ AC_INIT(export_vars.sh.in)
 AC_CONFIG_HEADER(include/private/apu_config.h)
 AC_CONFIG_AUX_DIR(build)
 
+LT_INIT
+
 sinclude(build/apu-conf.m4)
 sinclude(build/apu-iconv.m4)
 sinclude(build/apu-hints.m4)
